Components/Shipping/Providers/CartShippingCostsProvider.php

```diff
@@ -86,7 +86,11 @@ private function getInvoiceShippingCosts()
      */
     private function getTaxRate(Order $order)
     {
-        $taxRate = $order->getInvoiceShippingTaxRate();
+        $taxRate = null;
+
+        if (method_exists($order, 'getInvoiceShippingTaxRate')) {
+            $taxRate = $order->getInvoiceShippingTaxRate();
```

Comment:
what about DHL wunschpaket in that scenario?

Reply:
This doesn't affect DHL Wunschpaket.